deployments/container/reconfigure-mig.sh

```diff
@@ -599,7 +608,7 @@ if [ "${CDI_ENABLED}" = "true" ]; then
 		--dev-root=${DEV_ROOT_CTR_PATH} \
 		--vendor="management.nvidia.com" \
 		--class="gpu" \
-		--nvidia-ctk-path="/usr/local/nvidia/toolkit/nvidia-ctk" | \
+		--nvidia-cdi-hook-path=${NVIDIA_CDI_HOOK_PATH} | \
```

Comment:
This is a fairly recent edition. Is there a reason to use this instead of `nvidia-ctk cdi ...`?

Reply:
No particular reason. I just thought to pick up the new binary since it will be used moving forward. @elezar @klueska would you prefer if I stuck with `--nvidia-ctk-path` instead?

Reply:
For the record, this can be `nvidia-ctk` or `nvidia-cdi-hook` and the generated hook should be correct. 

For clarification: We want to separate the hook implementations from the `nvidia-ctk` to make them more easily auditable and / or consumable from non-OCI complaint environments such as LXD and/or Charliecloud for example.

Reply:
I don't have a real preference, but in newer versions of the container toolkit we prefer `nvidia-cdi-hook`. Can we be assured that users are using a v1.16.0 toolkit?

Reply:
We can't guarantee users are using v1.16.0 of the toolkit, but it will be the default version in the next operator release.